packages/office-ui-fabric-react/src/components/Dialog/DialogContent.styles.ts

```diff
@@ -16,13 +16,16 @@ const GlobalClassNames = {
 export const getStyles = (props: IDialogContentStyleProps): IDialogContentStyles => {
   const { className, theme, isLargeHeader, isClose, hidden, isMultiline, draggableHeaderClassName } = props;
 
-  const { palette, fonts, effects } = theme;
+  const { palette, fonts, effects, semanticColors } = theme;
 
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
   return {
     content: [
-      isLargeHeader && classNames.contentLgHeader,
+      isLargeHeader &&
+        classNames.contentLgHeader && {
```

Comment:
why check for contentLgHeader? Won't that always be true?

Reply:
@dzearing My mistake. My intent was to apply the contentLgHeader classname not use it as a check. I am pushing those changes in now, thanks!